components/scroll-wrapper/scroll-wrapper.js

```diff
@@ -56,7 +65,6 @@ class ScrollWrapper extends RtlMixin(LitElement) {
 			.d2l-scroll-wrapper-container {
 				box-sizing: border-box;
 				outline: none;
-				overflow-x: auto;
 				overflow-y: var(--d2l-scroll-wrapper-overflow-y, visible);
 			}
 			.d2l-scroll-wrapper-container:${unsafeCSS(getFocusPseudoClass())} {
```

Comment:
@dlockhart with your fix of the print styling, this is the only remaining style that isn't right - and I'm not sure a great way to fix it.

We want the `focus-visible` styles to be changed on the actual container, which with split-scrollers is some arbitrary element, rather than `.d2l-scroll-wrapper-container`. Unfortunately we can't add pseudo-selector styles via JS, and the element isn't guaranteed to be in the same DOM scope, so we can't fix it by e.g. just adding a class to the target.

Reply:
I've pushed some changes that address this in exactly the way you suggested.